backend/schema/corpora_schema_h5ad_implementation.md

```diff
@@ -0,0 +1,78 @@
+# Corpora Schema AnnData Implementation
+
+Corpora datasets are stored in the HDF5-backed AnnData format as written by version 0.7 of the anndata library. One
```

Comment:
we need some sort of top-level info in the H5AD to allow introspection on:
* what version of the corpora schema was encoded
* what version of this spec was encoded

I'd just a single semver field in uns that has something like `{ corpora_ver: 0.1, corpora_encoding: 0.4 }`

Reply:
Yeah agree, I will add that in.